app/src/main/java/com/google/droidjump/LevelsAdapter.java

```diff
@@ -49,12 +49,13 @@ public int getCount() {
 
     @Override
     public Object getItem(int index) {
+        // Receives index, returns level.
         return levels[index];
     }
 
-    @Override
-    public long getItemId(int index) {
-        return index - 1;
+    public long getItemId(int level) {
```

Comment:
the name of the method does not reflect from what the comment says. Maybe rename the method with a more appropriate name?

Reply:
That is a BaseAdapter method that even not used here but I can not remove it.

Reply:
Added @override keyword.